editor.planx.uk/src/lib/graphql.ts

```diff
@@ -37,8 +37,10 @@ const customFetch = async (
   return fetchResult;
 };
 
+console.log(import.meta.env.VITE_APP_HASURA_URL);
```

Comment:
This looks like a local testing remnant, can we remove the console log before merging?

Reply:
Good catch thanks, removed 🧹 🗑️